### ISSUE TITLE
Infrastructure appears not to be working

### ISSUE THREAD

**Issue by aantron** [ocsigen/ocsigen.org-data#13]
[Looking at the history](https://github.com/ocsigen/ocsigen.org-data/commits/master), there have been no commits by the bot since 27 July.

There is an Lwt release upcoming. I suppose it should work to generate the docs manually and open a PR for it.

**Comment by balat** [comment]
Indeed. I don't think we will fix that. We should probably switch everything to travis.
Contributions welcome ;)
